recipe/build_win.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+# Patch Rcpp
+# https://github.com/RcppCore/Rcpp/pull/1069
+sed -i -e 's;abs;std::abs;g' $BUILD_PREFIX/Lib/R/library/Rcpp/include/Rcpp/DataFrame.h
```

Comment:
Now that that Rcpp PR is merged, can you build a new version for conda and then delete this sed? Would this `sed` be safe once Rcpp with the fix is on conda, or would you end up with `std::std::abs`?